CONTRIBUTING.md

```diff
@@ -1,3 +1,4 @@
+
 # How to install and run the development version
 
```

Comment:
Added @language rest and @wrap to leojs.leo.

Hmm. Perhaps leojs.leo should have a "Startup" node, containing an @settings node.